### PR TITLE
FinancialInstitutionTransactionsService.update

### PR DESCRIPTION
**Pull request by cleclefibanity**

### CODE REVIEW

**Comment by damienbr** [comment]
:+1: I guess we need to wait for the release before merging

File: src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/UpdatedTransactionsServiceImpl.java
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/UpdatedTransactionsServiceImpl.java:44]: synchronizations -> synchronization

File: src/main/java/com/ibanity/apis/client/products/xs2a/mappers/SynchronizationMapper.java
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/xs2a/mappers/SynchronizationMapper.java:34]: if you update src/test/resources/json/synchronization.json, you can make sure that the parsing works properly in SyncrhonizationServiceImplTest.java

**Review by damienbr — CHANGES_REQUESTED** [review]

File: pom.xml
Comment (damienbr) [inline pom.xml:334]: :grimacing:

File: src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java
Comment (damienbr) [inline src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java:41]: Maybe you could add a test in TransactionsServiceImplTest -> @Test listUpdatedTransactions, in order to make sure we don't break this behavior in the future?

**Review by damienbr — APPROVED** [review]

**Review by damienbr — CHANGES_REQUESTED** [review]
Rollback of the pom required.

File: pom.xml
Comment (cleclefibanity) [reply pom.xml:334]: oups :-) Done

File: src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java
Comment (cleclefibanity) [reply src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java:41]: added

**Review by damienbr — APPROVED** [review]

**Pull request merged**
